### ISSUE TITLE
Duplicate Client Token Every Login

### ISSUE THREAD

**Issue by shane-rice-bfg** [evonove/django-oauth-toolkit#429]
I am using the framework, and it seems to work flawlessly, except every time the user logs in a second time, it duplicates the authorization in at the `o/authorized_tokens` endpoint.

Is there any way to prevent duplicate authorizations, or automatically remove the previous authorization upon a second login?

Currently the user can log in to all applications with out a password after the first authorization and as long as their logged into this oauth server, but each login create the duplicate client application, which is unsightly and hard to mange authorizations for the clients.

**Comment by jleclanche** [comment]
I have noticed this as well. I'd love a PR for it if you can, otherwise I'll take a crack at it.

**Comment by jleclanche** [comment]
@shane-rice-bfg have you had a chance to look into this?

**Comment by shane-rice-bfg** [comment]
No, I have not. We have moved away from OAuth for this application.

**Comment by bull500** [comment]
Hello @jleclanche 

Id like to give this a go
Any pointers/suggestions on how to approach this issue?

**Comment by n2ygk** [comment]
> Hello @jleclanche
> 
> Id like to give this a go Any pointers/suggestions on how to approach this issue?

@bull500 This is a really old bug report, so please make sure it still actually exists and feel free to submit a PR!

**Comment by bull500** [comment]
> @bull500 This is a really old bug report, so please make sure it still actually exists and feel free to submit a PR!

hello @n2ygk 

I did the Auth Code flow twice to test this.
The access tokens are different for me
![image](https://github.com/jazzband/django-oauth-toolkit/assets/8917275/269f6a97-1f8d-4c3e-a306-f9c8e0986e76)


On the http://127.0.0.1:8000/o/authorized_tokens/ page i do see the same app twice with individual token revoke links
![Screenshot from 2023-06-03 05-45-22](https://github.com/jazzband/django-oauth-toolkit/assets/8917275/494778a5-764f-45fc-928c-0baf22866652)

**Comment by dopry** [comment]
This is actually valid behavior believe it or not. It's up to the client to not hit the login endpoint again if they're already logged in. They may be logging in from multiple clients. The authorization endpoint doesn't have a way to know they already have a token.
